django_rq/stats_views.py

```diff
@@ -47,10 +64,12 @@ def stats(request):
     return render(request, 'django_rq/stats.html', context_data)
 
 
-def stats_json(request, token=None):
-    if request.user.is_staff or (token and token == API_TOKEN):
-        return JsonResponse(get_statistics())
+@never_cache
+def stats_json(request):
+    if not is_authorized(request):
+        return JsonResponse(
```

Comment:
@RasmusThing I'm not ready to break backward compatibility with the current `stats_json` view yet, I'd like to give users a warning and time to transition.

Do you mind creating a new view which maps to the url `/stats.json` with header token auth? This way we don't need to modify the current `def stats_json(request, token=None)` view.